### PR TITLE
feat: [PLG-1715]: Mutiny Integration 

### PR DESCRIPTION
**Pull request by devinder-harness**
Added `mutiny` to signup setup 
Context: We are integrating `mutiny` as a script in our Signup Screen for messaging A/B tests. Mutiny provides us with a unique persistent`visitorToken` that we used to track successful user conversion i.e. successful verified signup. We are passing this token to BE services so that it is passed to `NEW_SIGNUP` event and can be used in amplitude funnels to visualize user actiavtion on platorm
Script Info:
image

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -53,6 +55,19 @@
           window["adrum-config"] || (window["adrum-config"] = {})
         );
       }
+      if (window.isMutinyEnabled === "true" && mutinyToken) {
```
Comment (abhinavrastogi-harness) [inline index.html:58]: I'd suggest convert this to a global boolean. it'll be more maintainable. 
eg.
```
window.HARNESS_ENABLE_APPDY_EUM =  "HARNESS_ENABLE_APPDY_EUM_PLACEHOLDER" === "true";
```

File: index.html [hunk 2]
```diff
@@ -53,6 +55,19 @@
...
+        const mutinyScript = document.createElement("script");
+        mutinyScript.type = "text/javascript";
+        mutinyScript.text = `(function(){var a=window.mutiny=window.mutiny||{};if(!window.mutiny.client){a.client={_queue:{}};var b=["identify","trackConversion"];var c=[].concat(b,["defaultOptOut","optOut","optIn"]);var d=function factory(c){return function(){for(var d=arguments.length,e=new Array(d),f=0;f<d;f++){e[f]=arguments[f]}a.client._queue[c]=a.client._queue[c]||[];if(b.includes(c)){return new Promise(function(b,d){a.client._queue[c].push({args:e,resolve:b,reject:d});setTimeout(d,500)})}else{a.client._queue[c].push({args:e})}}};c.forEach(function(b){a.client[b]=d(b)})}})();`;
+        const mutinyClient = document.createElement("script");
```
Comment (abhinavrastogi-harness) [inline index.html:62]: can we make this async/defer? analytics/tracking scripts should never block the app load

File: src/static/images/ancestory.svg [hunk 3]
```diff
@@ -0,0 +1 @@
...1 1.032-2.77 1.032-3.393 0-.762-.482-1.038-1.926-1.108-.069 0-.069-.761 0-.761.963 0 2.202.069 3.027.069.619 0 1.239-.07 2.409-.07.068 0 .137.762 0 .762-.757.07-1.101.277-1.514.692-.551.554-.964 1.662-1.858 4.085v.001ZM20.844 4.894c-2.39-2.65-4.78-2.231-5.189-2.162.205.07.41 4.671 2.936 7.112 1.707 1.603 5.189 1.045 5.189 1.045s-.615-3.416-2.936-5.995Zm2.867 8.993c-.75 0-3.823.28-5.871 2.859-2.117 2.719-2.048 4.88-2.526 5.368.41.14 4.71.558 6.554-2.3 1.775-2.65 1.844-5.508 1.844-5.927ZM5.005 12.005c.478.07 2.117 2.23 5.053 2.51 3.208.348 4.3-1.325 4.37-1.325 0-.209-2.595-3.067-5.463-3.067-1.98.07-3.96 1.882-3.96 1.882Zm11.675 1.464c-.068.07-2.73 3.695-9.012 2.72C2.615 15.35.77 12.492.43 12.422c.41-.558 4.233-3.695 8.67-3.625 4.37.14 7.579 2.649 7.579 2.649h2.799c-2.253-.697-4.984-1.952-6.213-4.74-1.092-2.58-.887-4.114-1.365-6.275 0 0 5.052-.419 8.944 2.16 3.55 2.371 5.53 8.646 5.598 8.855h2.185L28.149.292l2.73.07c-.477 2.928.138 23.844 0 23.844-.272.07-2.594.208-2.798-.35 0 0 .546-10.248.614-10.457h-2.117c0 .14-.204 5.926-4.505 8.994-4.302 3.067-10.515 2.161-10.515 2.161.752-1.185 0-2.44 2.185-5.856 2.526-3.905 5.872-5.299 5.804-5.299-.205.07-2.8.07-2.868.07" fill="#4F5162"/></svg>
```
Comment (abhinavrastogi-harness) [inline src/static/images/ancestory.svg:1]: are the svg's optimised?
also can we check if the png will be smaller for these semi-complex logos?

File: index.html (hunk 2 above)
Comment (devinder-harness) [reply index.html:62]: no we cannot it as it will cause a flicker on screen

File: src/static/images/ancestory.svg (hunk 3 above)
Comment (devinder-harness) [reply src/static/images/ancestory.svg:1]: Yes let m check

Comment (devinder-harness) [reply src/static/images/ancestory.svg:1]: Yes these are optimized from svgo

**Review by kapilkumawat86 — APPROVED** [review]

**Review by vkbansal-harness — APPROVED** [review]

**Review by kapilkumawat86 — CHANGES_REQUESTED** [review]
Lets also add enable disable

**Review by kapilkumawat86 — APPROVED** [review]

**Review by abhinavrastogi-harness — APPROVED** [review]

**Review by kapilkumawat86 — APPROVED** [review]

**Pull request merged**